.github/workflows/deploy-preview.yml

```diff
@@ -21,46 +19,40 @@ jobs:
       - uses: pnpm/action-setup@v2
         with:
           version: 8
-      - name: Use Node.js 18
-        uses: actions/setup-node@v4
+      - uses: actions/setup-node@v4
         with:
           node-version: 18
           cache: "pnpm"
-      - name: Install dependencies
-        run: pnpm install
 
-      - name: Get branch name
+      - run: pnpm install
+
+      - name: Get cleaned up branch name
         id: branch-name
         uses: tj-actions/branch-names@v8
 
-      - name: Create Neon Branch
-        id: create-branch
+      - id: create-branch
         uses: neondatabase/create-branch-action@v4
         with:
           project_id: ${{ env.NEON_PROJECT_ID }}
           # parent: dev # optional (defaults to your primary branch)
           branch_name: preview/pr-${{ github.event.number }}-${{ steps.branch-name.outputs.current_branch }}
-          username: ${{ env.NEON_DATABASE_USERNAME }}
-          api_key: ${{ env.NEON_API_KEY }}
-
-      - name: Run Migrations
-        run: |
-          touch .env
+          username: ${{ secrets.NEON_DATABASE_USERNAME }} # change to your Neon database username / role
```

Comment:
I don't think `NEON_DATABASE_USERNAME` is actually secret worthy but I don't think this matters too much.

Reply:
Actually we didn't use a default value in the past, but now we default to `neondb_owner`, so might be worth removing this value

Reply:
Yes, let's do that.
```suggestion
```